src/gsp.py

```diff
@@ -58,6 +58,70 @@ async def get_all_available_forecasts(
     return forecasts
 
 
+# define route => /v0/solar/GB/gsp/forecast/{gsp_id}/{only_values} or other filter parameters
+# get latest gsp with only values
+# if getting latest
+#  async def get latest only values
+#    logger.info(f"Get forecasts for gsp id {gsp_id} with {forecast_horizon_minutes=}")
+
+# # return get_latest_forecast_values_for_a_specific_gsp_from_database(
+#     session=session,
+#     gsp_id=gsp_id,
+#     forecast_horizon_minutes=forecast_horizon_minutes,
+# )
+
+
+@router.get("/forecast/{gsp_id}/{only_values}", response_model=Union[Forecast, List[Forecast]])
```

Comment:
looks good, might be worth moving this `only_values' to a input, but not in the URL. I think its an option but shouldnt always be stated. 

Could rename to only_forecast_values